### ISSUE TITLE
First filter button often gets "X books found" text in screenreader

### ISSUE THREAD

**Issue by seanredmond** [NYPL/staff-picks#137]
When tabbing with Chrome and VoiceOver on from the "X books found" message to the first filter button, the text of the button is is not read. Instead the "X books found" is read instead. If you are shift-tabbing in the other direction, the first filter label is read fine.

Watching what's displayed on the screen by VoiceOver, I can see that , for a split second, it's about to say the button label but that it gets overriden by the books found. I wonder if focusing the button is triggering an event that make VoiceOver think there has been a change that aria-live should announce even thought he change is from 50 books to 50 books? 

Not sure if this is related to #118 or not.

**Comment by courtneymcgee** [comment]
@wlla , @seanredmond references #118 (Inaccurate list of tags displayed with Chrome and Jaws) above, but might this be a result of #105 (Once a filter is applied, focus should go to top of list), which we put in place to address #101 (Aria-live region is not announcing the change in the number of books)? Apologies for all of the references! I'd love your opinion, as I'm trying to make a call on if we can or will address.
